config/settings/development.py

```diff
@@ -0,0 +1,3 @@
+from .common_settings import * # noqa
```

Comment:
Why the relative import here? We're not using them anywhere else in the project.